src/RestControllers/AuthorizationController.php

```diff
@@ -114,15 +114,16 @@ public function __construct($providerForm = true)
                 http_response_code(500);
                 exit;
             }
-            // second, create and store the private/public key pain
-            $keys = openssl_pkey_new(["private_key_bits" => 2048, "private_key_type" => OPENSSL_KEYTYPE_RSA]);
+            // second, create and store the private/public key pair
```

Comment:
lol, pain :)

Reply:
yep, looks like an xampp thing but I believe windows server have the same issue.